tests/python/test_internal_func.py

```diff
@@ -9,7 +9,7 @@ def all_archs_for_this(test):
     return ti.archs_excluding(ti.metal, ti.opengl, ti.cuda, ti.vulkan)(test)
 
 
-@ti.test(exclude=[ti.metal, ti.opengl, ti.cuda, ti.vulkan])
+@ti.test(exclude=[ti.metal, ti.opengl, ti.cuda, ti.vulkan, ti.cc])
```

Comment:
would you mind elaborating on why we are disabling these tests on cc backend? Is it only for a certain opt_level or for both? :D

Reply:
cc backend is not in a maintained mode (but still serves as a nice codegen reference), we don't have to let it get in our way.

Reply:
@k-ye yea I totally agree, I guess I'm just curious why it failed and we should make sure it's understood and added in the comment ;)

Reply:
cc backend doesn't have functions from taichi/runtime/llvm/internal_functions.h